components/802.15.4_RF/stm-s2lp-rf-driver/source/NanostackRfPhys2lp.cpp

```diff
@@ -216,7 +222,7 @@ static const phy_device_channel_page_s phy_channel_pages[] = {
 };
 
 #ifdef MBED_CONF_RTOS_PRESENT
-#include "mbed.h"
+
```

Comment:
```suggestion
#include "Thread.h"
using namespace rtos;
```

Move using namespace rtos inside ` #ifdef MBED_CONF_RTOS_PRESENT`